ui/main.qml

```diff
@@ -549,6 +567,19 @@ StatusWindow {
         }
     }
 
+    Component {
+        id: keycard
+        Keycard {
```

Comment:
Maybe we can give this a more semantic name?

Reply:
I first called it KeycardOnboarding but then I noticed I am already in the Onboarding directory so I wanted to avoid redudancy. Do you think KeycardOnboarding would be a better name or should I think of something else?